tools/system_libs.py

```diff
@@ -1859,6 +1859,10 @@ def install_system_headers(stamp):
   bin_dest = shared.Cache.get_sysroot_dir('bin')
   copytree_exist_ok(bin_src, bin_dest)
 
+  cmake_src = utils.path_from_root('system/cmake')
+  cmake_dest = shared.Cache.get_sysroot_dir('cmake')
+  copytree_exist_ok(cmake_src, cmake_dest)
```

Comment:
How does cmake know to look in `<sysroot>/cmake` for this file?

On my system the `cmake` files for SDL seem to be in the `lib` directory:

```
$ dpkg -L libsdl2-dev | grep cmake
/usr/lib/x86_64-linux-gnu/cmake/SDL2/sdl2-config-version.cmake
/usr/lib/x86_64-linux-gnu/cmake/SDL2/sdl2-config.cmake
```

Should we follow suite and install these files into `<libdir>/cmake/SDL2` which would be `<sysroot>/lib/wasm32-emscripten/cmake/SDL2`?

Reply:
It searches in quite a few paths: https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure. `<sysroot>/lib/wasm32-emscripten/cmake/SDL2` would also work.

Reply:
I think maybe `<sysroot>/lib/cmake/` would be the most appropriate.. and avoid having to duplicate it for `wasm64-emscripten`.